README.md

```diff
@@ -1,176 +1,189 @@
-     If you are using csh or tcsh, then use this line instead:
-        setenv PATH /home/luser/bin:${PATH}
-     By prepending your home directory to the rest of the PATH you can
-     override systemwide installed software with your own custom installation.
-
-
-CONFIGURATION
-
-     See:
-     https://www.ibm.com/support/knowledgecenter/linuxonibm/com.ibm.linux.z.lxce/lxce_stackoverview.html
-
-     Prior to version 3, opencryptoki used pk_config_data as its
-     configuration file. This file was created upon running pkcs11_startup.
-     In version 3, pkcs11_startup and pk_config_data have been
-     removed and replaced with a customizeable config file named,
-     opencryptoki.conf. It contains an entry for each token currently
-     supported by opencryptoki. However, only those tokens, whose
-     hardware and software requirements are available on the local system,
-     will show up as present and available upon running the pkcsconf -t
-     command.
-
-     Before using, each token must be first initialized.
-     You can select the token with the -c command line option; refer
-     to the documentation linked to above for further instructions.
-
-     Initialize a particular token by running pkcsconf:
-       % pkcsconf -I -c
-
-     In this version of openCrypoki, the default SO PIN is 87654321.
-     This should be changed to a different PIN value before use.
-
-     You can change the SO PIN by running pkcsconf:
-       % pkcsconf -P -c
-
-     You can initialize and change the user PIN by typing:
-       % pkcsconf -u -c
-
-     You can later change the user PIN again by typing:
-      % pkcsconf -p -c
+Please see [ChangeLog](ChangeLog) for release specific information.
+
+## OVERVIEW
+
+OpenCryptoki version 3.7.0 implements the PKCS#11 specification version 2.20.
+
+This package includes several cryptographic tokens:
+CCA, ICA, TPM , SWToken, ICSF, and EP11.
+
+## REQUIREMENTS:
+
+- IBM ICA - requires libica library version 2.3.0 or higher for accessing ICA hardware crypto on IBM
+  zSeries.
+- IBM CCA - requires IBM XCrypto CEX3C card and the
+- TPM     - requires a TPM, TPM tools, and TCG software stack.
+- SWToken - The software token uses OpenSSL version 0.9.7 or higher.
+- ICSF    - The Integrated Cryptographic Service Facility (ICSF) token requires openldap and
+  openldap client software version 2.4.23 or higher. Lex and Yacc are also required to build this
+  token.
+- EP11    - The EP11 token is a token that uses the IBM Crypto Express adapters (starting with
+  Crypto Express 4S adapters) configured with Enterprise PKCS#11 (EP11) firmware.
+
+## BUILD PROCESS
+
+The simplest way to compile this package is to enter the source code main directory and do the
+following:
+
+1. Run the bootstrap.sh script by typing:
+```
+    $ ./bootstrap.sh
+```
+
+2. Configure the source code by typing:
+```
+    $ ./configure
+```
+
+If you're planning to install the package into your home directory or to a location other than
```

Comment:
You need add a TAB here to continue the enumeration of the steps. Without this TAB, this would be considered a new paragraph and the item 3 below will be printed as 1 again.

s/+If/+   If/